release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -580,6 +580,21 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* Machines not prefixed with the infrastructure ID are preserved when destroying
```

Comment:
The firewall rules used by the machines are preserved not the machines themselves.